odoo/addons/base/models/ir_module.py

```diff
@@ -142,7 +142,7 @@ def get_transforms(self):
 
 STATES = [
     ('uninstallable', 'Uninstallable'),
-    ('uninstalled', 'Not Installed'),
+    ('uninstalled', 'Uninstalled'),
```

Comment:
Are you sure for this change ? As this is default state, you may think something has been removed although it was just not installed. Seems strange.